examples/05_graphical_pipelines.ipynb

```diff
@@ -0,0 +1,5146 @@
+    "Many tasks are non-sequential. To solve this two possibilities exist:\n",
+    "1. Nesting Sequential Pipelines.\n",
+    "2. Using Graphical Pipelines.\n",
+    "\n",
+    "\n",
+    "Thus, there is the generalised graphial pipeline.\n",
+    "* Graphical means that different steps may share the same predecessor or provide their outputs to the same successor (the dataflows can branch and merge).\n",
+    "<img src=\"img/graphical_pipeline.png\" width=750 />\n",
+    "\n",
+    "\n",
+    "* Generalised means that the pipeline can be used for multiple tasks (e.g. forecasting, classification, ...).\n",
+    "\n",
+    "**Note**\n",
+    "\n",
+    "The graphical pipeline is still experimental. Thus, this graphical should not used in production. However, we would be happy to get feedback on the graphical pipeline.\n",
+    "\n",
+    "\n",
+    "\n",
+    "### Potential Use-Cases\n",
+    "There exist various potential use-case for the graphical pipeline. In the following, we focus on a forecasting and a classification pipeline.\n",
+    "#### Forecasting Use-Case for Graphical Pipelines\n",
+    "\n",
+    "\n",
+    "The input of forecasters depends on the output of other forecasters, which same the same input.\n",
+    "* Forecaster could use the same preprocessing (branching of data flow)\n",
+    "* Forecaster could use outputs of multiple predeccessors (merging of data flow)\n",
+    "\n",
+    "<img src=\"img/graphical_pipeline_example.png\" width=900 />\n",
+    "\n",
+    "\n",
+    "**Note:** The current experimental state of the graphical pipeline does not fully support this use-case. However, we are working on this. If you are interested in this use-case and want to contribute, please contact us.\n",
+    "\n",
+    "### Credits\n",
+    "The graphical pipeline was first developed by pyWATTS [1] and was then adapted for sktime. The original implementation can be found [pyWATTS](https://github.com/KIT-IAI/pyWATTS). pyWATTS is a open source library developed at the Institute of Applied Informatics and Automation at the KIT and funded by HelmholtzAI.\n",
+    "\n",
+    "> [1] Heidrich, Benedikt, et al. \"pyWATTS: Python workflow automation tool for time series.\" arXiv preprint arXiv:2106.10157 (2021)."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "collapsed": false
+   },
+   "source": [
+    "\n",
+    "**Note:** The current experimental state of the graphical pipeline does not fully support this use-case. However, we are working on this. If you are interested in this use-case and want to contribute, please contact us.\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "collapsed": false
+   },
+   "source": [
+    "## How to build a Graphical Pipeline\n",
+    "Two ways to build a graphical pipeline:\n",
+    "\n",
+    "1. Pass all steps to the pipeline during initialisation as for the sequential pipelines.\n",
+    "\n",
+    "```python\n",
+    "pipeline = Pipeline([\n",
```

Comment:
should this code not be in jupyter cells? Users will attempt to copy or execxute these, but it is not possible. Imo notebooks like this work best if the instructions are followed by actual code.

Reply:
I see why you don't use actual code - you want to explain it abstractly.
Trying to read it with the mindset of someone new to this, I would be confused about (a) what is `skobject1`, and (b) the lack of reference conceptualization (e.g., the graph). Perhaps add a simple example, or merge it with the first?

Reply:
I merged it with the following example. Should now be easier to understand